internal/encrypt/encrypt.go

```diff
@@ -88,13 +88,20 @@ func (e *Encrypter) Encrypt(reader io.Reader, filename string) error {
 	return nil
 }
 
+const outDirName = "out"
+
 func (e *Encrypter) generateAndSaveMasterKey(filename string) ([]byte, error) {
 	masterKey, err := sss.GenerateMasterKey()
 	if err != nil {
 		return nil, err
 	}
 
-	err = file.WriteKey(masterKey, "out/"+filename)
+	if err := os.MkdirAll(outDirName, 0744); err != nil {
```

Comment:
Is `0644` enough?

Reply:
You need the 'execute' bit to traverse the dir and access its content, I think.

Reply:
That's correct, it works for a single dir but it fails for multiple subdirectories!